L1Trigger/L1TMuonEndCap/src/AngleCalculation.cc

```diff
@@ -90,16 +89,12 @@ void AngleCalculation::calculate_angles(EMTFTrack& track) const {
       }
     }
 
-    if (bugME11Dupes_) {
-      if (not (st_conv_hits.at(istation).size() <= 4)) // ambiguity in theta is max 4
-	{ edm::LogError("L1T") << "st_conv_hits.at(istation).size() = " << st_conv_hits.at(istation).size(); return; }
-    } else {
-      if (not (st_conv_hits.at(istation).size() <= 2))  // ambiguity in theta is max 2
-	{ edm::LogError("L1T") << "st_conv_hits.at(istation).size() = " << st_conv_hits.at(istation).size(); return; }
-    }
+    if (bugME11Dupes_)
```

Comment:
Pretty sure we're not allowed to have any "assert" statements - all errors have to be logged.